.github/workflows/build-preview.yml

```
@@ -0,0 +1,47 @@
+name: Build Preview
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  build-preview:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+      - run: npm install
```

```suggestion
      - run: npm ci
```

I think this might be a good idea in this case. The way I understand it is, that this will skip evaluating the dependencies and just install what is provided by the package-lock, which is faster. And in this case, fast and "the same way as before" is good.

Did a quick google and articles like https://www.voorhoede.nl/en/blog/super-fast-npm-install-on-github-actions/  suggest that is true.

---

`npm ci` requires the `package-lock.json` to be committed into the repo, which has not been the case until now. I don't see a reason why we shouldn't, though. So, `clean-install` it is with 16fc132.

---

ps: the `npm install` for the iD repo is still necessary until the iD repo also includes the `package-lock.json` file.